payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AuBankAccountNumberSpec.kt

```diff
@@ -0,0 +1,12 @@
+package com.stripe.android.ui.core.elements
+
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+internal object AuBankAccountNumberSpec : SectionFieldSpec(IdentifierSpec.Generic("account_number")) {
+    fun transform(): SectionFieldElement =
+        AuBankAccountNumberElement(
```

Comment:
I believe we can just make this a SimpleTextElement rather than the specific type of AuBankAccountNumberElement, but double check me on this.